manual/pyscf_for_afqmc.tex

```diff
@@ -0,0 +1,132 @@
+\section{Using PYSCF to generate integrals for AFQMC}
+\label{sec:pyscf}
+These notes describe useful knowledge and tips in the use of pyscf. 
```

Comment:
'PySCF' is the capitalization used in their documentation.   It should be consistently written everywhere in this section.

Also, it might be useful to give the URL for the PySCF project.

Reply:
Will fix soon.

Reply:
Fixed